app/src/main/java/de/westnordost/streetcomplete/quests/smoothness/AddRoadSmoothness.kt

```diff
@@ -58,6 +58,12 @@ class AddRoadSmoothness : OsmFilterQuestType<SmoothnessAnswer>() {
                 changes.deleteIfExists("smoothness:date")
                 changes.deleteCheckDatesForKey("smoothness")
             }
+            is IsActuallyStepsAnswer -> {
```

Comment:
a road is actually steps? I don't think we want to enable such an answer for `AddRoadSmoothness`

Reply:
I agree, it should have been caught much before one got to smoothness quest, or an OSM note left instead if that is really the case.